docs/reference/tools-reference/llm-tool.md

```diff
@@ -158,3 +158,75 @@ In LLM tool, the prompt is transformed to match the [openai messages](https://pl
     }
 ]
 ```
+
+### Sample 3
+```jinja
+# system:
```

Comment:
The example looks a bit complicated, could we simplify it based on the purpose? I think we just want to teach how to compose role 'tool'. Special thing for role 'tool' we find is it must be used with 'tool_calls'.

And please also add some comments to instruct user why must write like this.

Reply:
updated, please help to take another look